MVC_Activity01/MVC_Activity01/Views/Customers/Edit.cshtml

```diff
@@ -1,7 +1,7 @@
-﻿@model MVC_Activity01.Models.Customers
+﻿@model MVC_Activity01.Models.Customer
 
 
-@using (Html.BeginForm("Edit", "Customer", FormMethod.Post))
+@using (Html.BeginForm("Edit", "Customers", FormMethod.Post))
 {
     @Html.HiddenFor(m => m.Id)
     <form cellpadding="0" cellspacing="0" class="container">
```

Comment:
You already have a Html.BeginForm you don't need another form.

Reply:
ok sir @jrvalenzona-bps  thank you sir for the Info.